python/cucim/tests/unit/clara/test_load_image_metadata.py

```diff
@@ -45,7 +45,7 @@ def test_load_image_metadata(testimg_tiff_stripe_32x24_16):
     # Returns physical size in tuple.
     assert img.spacing() == [1.0, 1.0, 1.0]
```

Comment:
to test a bit more thoroughly, could you also add something like `resolution=(0.5, 2.0, "CENTIMETER"),` as a kwarg to the TiffWriter's `write` call in tests/util/gen_tiff.py?

`tifffile`'s resolution argument takes two floating point or integer resolution values and optionally a unit as described [here](https://github.com/cgohlke/tifffile/blob/375d97f62df6482142b51f1b38a49bdd24d18a60/tifffile/tifffile.py#L1629-L1633)

I checked that `tifffile` writes to the tags you have implemented here: https://github.com/cgohlke/tifffile/blob/375d97f62df6482142b51f1b38a49bdd24d18a60/tifffile/tifffile.py#L2557-L2571

Reply:
Thanks @grlee77 for the suggestion and relevant information! (I was seeking such information on the usage of tifffile so Thank you!)
I would work on it tomorrow.